docs/guide/slots.md

```diff
@@ -107,18 +115,22 @@ end
 
 ## Lambda slots
 
-It's also possible to define a slot as a lambda that returns content to be rendered (either a string or a ViewComponent instance). Lambda slots are useful for working with helpers like `content_tag` or as wrappers for another ViewComponent with specific default values:
+Slots can be defined as lambdas that returns content to be rendered. Lambda slots are useful when you're outputting
+basic markup that doesn't warrant its own component yet.
```

Comment:
IMO this loses some of the clarity and specificity of the original text. Maybe we could:

```suggestion
It's also possible to define a slot as a lambda that returns content to be rendered (either a string or a ViewComponent instance). Lambda slots are useful in cases where writing another component may be unnecessary, such as working with helpers like `content_tag` or as wrappers for another ViewComponent with specific default values:
```